WordPress/Tests/DB/RestrictedFunctionsUnitTest.php

```diff
@@ -46,7 +46,6 @@ public function getErrorList() {
 			48 => 1,
 			49 => 1,
 			50 => 1,
-			51 => 1,
```

Comment:
This change was made in the wrong file (which is why the unit tests are still failing).
You need to change the `WordPress/Tests/VIP/RestrictedFunctionsUnitTest.php` file. This is the database restricted unit test file.